shopelectro/models.py

```diff
@@ -82,6 +82,8 @@ class Product(
         verbose_name=_('tags'),
     )
 
+    # @todo #910:30m  Explore and doc what is vendor_code.
```

Comment:
I still remember this story :) The root of evil here 1C integration

Reply:
@artemiy312 , then it's task for you =)